src/overlays/actors/ovl_Obj_Flowerpot/z_obj_flowerpot.c

```diff
@@ -428,8 +428,11 @@ void ObjFlowerpot_Init(Actor* thisx, GlobalContext* globalCtx) {
     D_80A1D400++;
 }
 
-void ObjFlowerpot_Destroy(Actor* thisx, GlobalContext* globalCtx) {
-    Collider_DestroyJntSph(globalCtx, &THIS->collider);
+void ObjFlowerpot_Destroy(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    ObjFlowerpot* this = THIS;
+
+    Collider_DestroyJntSph(globalCtx, &this->collider);
 }
```

Comment:
This is an interesting one, because it's not clear that this function actually needs the recast: it's quite possible that all these collision functions use `gameState`.

Reply:
Yeah, its either this, or use THIS as before.

Reply:
As in, it will match with both, or neither, but not just `ObjFlowerpot* this = THIS;`?

Reply:
Yeah. Its either use both, or use `Collider_DestroyJntSph(globalCtx, &THIS->collider);`